expo-maps/android/src/main/java/expo/modules/maps/PropsData.kt

```diff
@@ -107,3 +107,34 @@ data class ClusterObject(
 ) : Record
 
 data class KMLObject(@Field val filePath: String) : Record
+
+data class GeoJsonObject(
+  @Field val geoJsonString: String,
+  @Field val defaultStyle: GeoJsonObjectDefaultStyle?
+) : Record
+
+data class GeoJsonObjectDefaultStyle(
+  @Field val polygon: GeoJsonObjectDefaultStylePolygon?,
+  @Field val polyline: GeoJsonObjectDefaultStylePolyline?,
+  @Field val marker: GeoJsonObjectDefaultStyleMarker?
+) : Record
+
+data class GeoJsonObjectDefaultStylePolygon(
+  @Field val fillColor: String?,
+  @Field val strokeColor: String?,
+  @Field val strokeWidth: Float?,
```

Comment:
I have a question here. Why did you use `Float` as a type here? I'm curious, cause the more natural representation of the js number in `Kotlin` will be `Double`.

Reply:
In that case no, if the library below uses `float`, let's stick to it ;)